src/oc_erchef/apps/chef_license/src/chef_license_worker.erl

```diff
@@ -28,7 +28,7 @@
     message
 }).
 
--define(DEFAULT_LICENSE_SCAN_INTERVAL, 30000). %milli seconds
```

Comment:
@kalroy What if the new license applied. This blocks the customer's who wants to refresh immediately .